huaweicloud/services/acceptance/gaussdb/resource_huaweicloud_gaussdb_redis_instance_test.go

```diff
@@ -51,13 +63,19 @@ func TestAccGaussRedisInstance_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "status", "normal"),
 				),
 			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"availability_zone", "password"},
+			},
 		},
 	})
 }
 
 func testAccCheckGaussRedisInstanceDestroy(s *terraform.State) error {
-	config := acceptance.TestAccProvider.Meta().(*config.Config)
-	client, err := config.GeminiDBV3Client(acceptance.HW_REGION_NAME)
+	cfg := acceptance.TestAccProvider.Meta().(*config.Config)
+	client, err := cfg.GeminiDBV3Client(acceptance.HW_REGION_NAME)
 	if err != nil {
 		return fmtp.Errorf("Error creating HuaweiCloud GaussRedis client: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("error creating GaussRedis client: %s", err)
```
don't forget to udpate other error messages in this file

Reply:
this PR does not fix lint issue